docs/.vitepress/theme/custom.css

```diff
@@ -3,6 +3,7 @@
   --vp-c-brand-2: #c63f36;
   --vp-c-brand-3: #CF4840;
   --vp-c-brand-soft: rgba(255,72,64,.14);
+  --vp-z-index-nav: 20;
```

Comment:
I think you need to remove this line. I investigated together with @magnuh and he found that we need to update the package.json with the latest versions of @warp-ds/css and @warp-ds/vue. However, we need to make a new stable release of the @warp-ds/vue and we also need the latest commit from the vue-repo to be included in that next stable version, since that one bumps the @warp-ds/css package. Currently the latest commit is a chore and therefore hasn't gotten a semantic-release-tag. @magnuh is working on updating the @warp-ds/vue's next stable release. Once that is done, you just need to bump to the latest version. :)

Reply:
Reverted 👍